### PR TITLE
Fixing building from source instructions in README

### PR DESCRIPTION
**Pull request by nirajpatel**

### CODE REVIEW

**Review by asimshankar — COMMENTED** [review]
Thanks for the update. Quick note.

File: tensorflow/java/README.md [hunk 1]
```diff
@@ -120,8 +120,8 @@ libraries will need to be built from source.
       //tensorflow/java:libtensorflow_jni
     ```
 
-The JAR (`libtensorflow.jar`) and native library (`libtensorflow_jni.so`) will 
-be in `bazel-bin/tensorflow/java`.
+The JAR (`libtensorflow.jar`) and native library (`libtensorflow_jni.dylib`) will
```
Comment (nirajpatel) [inline tensorflow/java/README.md]: Hey @asimshankar! Bazel creates the `dylib` file as well which is needed in order to run any of the logic in the tensorflow jar that calls out to the native code (see https://github.com/tensorflow/tensorflow/tree/master/tensorflow/java#quickstart step 4). I am not sure what the `libtensorflow.so` file does. If you could find out on your end that would be helpful! I don't think I am using it though (running on my mac).

Comment (nirajpatel) [inline tensorflow/java/README.md]: Oh, so it seems like you can use either/or. Is that correct?
https://www.chilkatsoft.com/java-loadLibrary-Linux.asp

**Comment by nirajpatel** [comment]
Fixed @asimshankar! Please check again now.

File: tensorflow/java/README.md [hunk 2]
```diff
@@ -120,8 +122,8 @@ libraries will need to be built from source.
       //tensorflow/java:libtensorflow_jni
     ```
 
-The JAR (`libtensorflow.jar`) and native library (`libtensorflow_jni.so`) will 
-be in `bazel-bin/tensorflow/java`.
+The JAR (`libtensorflow.jar`) and native library (unix: `libtensorflow_jni.so` or mac os x:`libtensorflow_jni.dylib`) will
```
Comment (asimshankar) [inline tensorflow/java/README.md]: Let's use the terms and formatting used in the rest of the doc:

(`libtensorflow_jni.so` on Linux and `libtensorflow_jni.dylib` on OS X)

File: tensorflow/java/README.md [hunk 3]
```diff
@@ -85,6 +85,8 @@ Java bindings for TensorFlow. ([Javadoc](https://www.tensorflow.org/api_docs/jav
       org.tensorflow.examples.LabelImage
     ```
 
+    Note: you can also dynamically link the native library at runtime by using `System.load()`.
```
Comment (asimshankar) [inline tensorflow/java/README.md]: I'd suggest removing this. It's a hard line to draw between keeping the README short and focused enough for the common use case and covering all possibilities. In this particular case, I'd prefer omitting this.

**Comment by nirajpatel** [comment]
@asimshankar fixed!

**Review by asimshankar — APPROVED** [review]
Thanks!

**Comment by jhseu** [comment]
Jenkins, test this please

**Pull request merged**